model/plugins/update_audit/structures.smithy

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+structure AuditConfig{
+    enabled: Boolean
+    audit: AuditLogs
+    compliance: Compliance
+}
+
+@input
+structure PutAudit_Input{
+    @required
+    @httpPayload
+    content: AuditConfig
+}
+
+@output
+structure PutAudit_Output {
+    content: Response
```

Comment:
The output body doesn't use `content` because it's meant to be used along with `@httpPayload`. Instead put the body members directly into this structure like in the [search action](https://github.com/opensearch-project/opensearch-api-specification/blob/main/model/_global/search/structures.smithy)